.buildkite/scripts/unit_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+source .buildkite/scripts/install_tools.sh
+
+set -euo pipefail
+
+echo "--- Run Unit Tests"
+
+#sudo command doesn't work at the "pre-command" hook because of another user environment (root with strange permissions)
+sudo chmod -R go-w "${BEATS_PROJECT_NAME}/"     #fix the fulesystem permissions issue like this:https://buildkite.com/elastic/beats-metricbeat/builds/1154#018d12db-dc0c-4bcd-b9b4-d5dece0b42c6/272-1267
```

Comment:
Can you raise a GitHub issue with all these details to assign it to the relevant team?

Reply:
Has an issue been created for this?